src/pages/Autolending/LightboxFilter.vue

```diff
@@ -17,7 +17,7 @@
 			radio-value="some"
 			v-model="radio"
 		>
-			Specific {{ name }} only.
+			Specific {{ name }} only
```

Comment:
Leave this for now as the issue is still being discussed https://kiva.atlassian.net/browse/CASH-1490

Reply:
Thanks @emuvente I didn't know this discussion was happening in another ticket. Removed this change from this pull request.